src/include/simeng/models/emulation/Core.hh

```diff
@@ -67,11 +76,22 @@ class Core : public simeng::Core {
   /** Handle an encountered exception. */
   void handleException(const std::shared_ptr<Instruction>& instruction);
 
-  /** Process an active exception handler. */
-  void processExceptionHandler();
+  /** Process the active exception. */
+  void processException();
+
+  /** Create an instance of the exception handler based on the chosen
+   * architecture. */
+  void exceptionHandlerFactory(std::string isa) {
```

Comment:
Given that all functions haven't been defined in the header, just to be consistent, could you please move the definition of this method to the corresponding .cc file

Reply:
Oh wait I misread your comment

Reply:
Resolved offline